swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -171,12 +135,11 @@ type Doctor record {|
     decimal fee;
 |};
 
-configurable int port = 8290;
 configurable string healthcareBackend = "http://localhost:9090/healthcare";
 
 final http:Client queryDoctorEP = check new (healthcareBackend);
 
-service /healthcare on new http:Listener(port) {
+service /healthcare on new http:Listener(8290) {
     resource function get doctors/[string category]() 
             returns Doctor[]|http:NotFound|http:InternalServerError {
         log:printInfo("Retrieving information", specialization = category);
```

Comment:
Can you also update 

"Define a variable to keep the expected payload from the backend service" under testing to say something like

"Assign the mock payload from the backend service to a variable..."

"to keep" isn't ideal IMO.